service/src/twin.ts

```diff
@@ -7,7 +7,8 @@ import { errors } from 'azure-iot-common';
 import * as _ from 'lodash';
 import { DeviceIdentity } from './device';
 import { Registry } from './registry';
-import { Callback } from './interfaces';
+import { IncomingMessageCallback, ResultWithIncomingMessage, createResultWithIncomingMessage } from './interfaces';
+import { tripleValueCallbackToPromise } from 'azure-iot-common';
```

Comment:
>import  [](start = 0, length = 7)

Nit: one import from the azure-iot-common module?

Reply:
Done.

---
In reply to: [223081020](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r223081020) [](ancestors = 223081020)